Sources/AppStoreConnectCLI/Model/BuildDetailsInfo.swift

```diff
@@ -5,6 +5,7 @@ import Foundation
 import SwiftyTextTable
 
 struct BuildDetailsInfo: ResultRenderable {
+  let id: String
```

Comment:
For our models we don't really want to expose the internal ids
`ReadBuildOperation` should return the SDK model so that you can access the `id`
Conversion to `BuildDetailsInfo` can be done as needed in the service